src/desktop/apps/fair/templates/sections.jade

```diff
@@ -1,20 +1,18 @@
 .fair-sections-container
   .sections-left
     .section-left-info-nav
-      h2 Fair Information
+      h2 Information
       .fair-nav-info-menu.chevron-nav-list
         a(href="/#{profile.id}/info/visitors") visitors
         if infoMenu.programming
           a(href="/#{profile.id}/info/programming") programming
         if infoMenu.events
           a(href="/#{profile.id}/info/events") events
         if infoMenu.artsyAtTheFair
-          a(href="/#{profile.id}/info/artsy-at-the-fair") artsy at the fair
-        a(href="/#{profile.id}/info/about-the-fair") about the fair
+          a(href="/#{profile.id}/info/artsy-at-the-fair") artsy at the event
```

Comment:
This one is just some more tabs that barely ever displayed seems like but seems like removing Fair is what we wanted there as well.